updated kafka flow (#659)
redhat-developer/app-services-guides: docs/kafka/rhoas-cli-getting-started-kafka/README.adoc

.Prerequisites

* You're logged in to `rhoas`.

.Procedure

. Enter the following command to create a Kafka instance with default values.
+
--
.Creating a Kafka instance
[source,shell]
----
$ rhoas kafka create
----
--

> . Enter Name: `my-kafka`.

Reviewer: ```suggestion
. Enter `my-kafka` to create a Kafka instance called `my-kafka`.
```

> . Use the arrows on your keyboard to select your OpenShift cluster for your Kafka instance.

Reviewer: ```suggestion
. The CLI prompts you to select a cluster to provision your Kafka instance on. Use the arrows on your keyboard to select the required cluster.
```

> . Select an instance size of 1 (x1) or 2 (x2) streaming units. 

Reviewer: ```suggestion
.  The CLI prompts you to select an instance size. Use the arrows on your keyboard to select an instance size.
```

+
--
A streaming unit determines the default maximum capacity of a Kafka instance. An instance with a larger size can handle higher loads and process more events, has more storage, and can handle more clients and connections.

> The CLI displays information about the created instance including billing model, cloud provider, cluster id of the cluster the instance is provisioned on, and instance type.

Reviewer: ```suggestion
The CLI displays information about the created instance including billing model, cloud provider, cluster ID of the cluster the instance is provisioned on, and instance type.
```

CLI uses service contexts to group service instances against which commands are executed.
The created Kafka instance is set automatically in the current context.

[NOTE]
====

> Alternatively, to provision a Kafka instance by specifying the kafka name and cluster-id, enter the following command: `rhoas kafka create --name my-kafka --cluster-id <CLUSTER_ID>`. This takes on the default value size 1 streaming unit.

Reviewer: ```suggestion
Alternatively, to provision a Kafka instance by specifying the Kafka name and cluster ID, enter the following command: `rhoas kafka create --name my-kafka --cluster-id <CLUSTER_ID>`. Replace <CLUSTER_ID> with your cluster ID. Using this command creates an instance size with the default value size of 1 streaming unit.
```

Reviewer: @fmcdonal. As per [user-replaced values guidelines](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values),  the value needs to be italicized. So the command should be: `rhoas kafka create --name my-kafka --cluster-id _<CLUSTER_ID>_`.

Reviewer: Hi @fmcdonal. This value needs to be italicized.

====
--
. Verify that the Kafka instance has a `ready` status.
+
--
[source,shell]
----
$ rhoas kafka list
----

> .Reviewing details of a Kafka instance

Reviewer: There are two commands here now. If the `context status` command is still used, then it needs to be separated out from the `kafka list` one to avoid confusion.
